src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -464,13 +492,88 @@ def example_repl(self, text, example, start_index, continue_flag):
                         start_index += 1
                         cmd += " " + answer.split()[-1] + " " +\
                                u' '.join(text.split()[start_index:start_index + 1])
+            self.completer.set_scenario_enabled(True)
             example_cli.exit()
             del example_cli
         else:
             cmd = text
 
         return cmd, continue_flag
 
+    def handle_scenario(self, text):
+        """ parses for the scenario recommendation """
+        num = text.partition(SELECT_SYMBOL['example'])[2].strip()
+        try:
+            num = int(num) - 1
+        except ValueError:
+            print("An Integer should follow the colon", file=self.output)
+            return
+        if 0 <= num < len(self.recommender.get_scenarios() or []):
+            scenario = self.recommender.get_scenarios()[num]
+            self.recommender.feedback_scenario(num, scenario)
+        else:
+            print('Invalid example number', file=self.output)
+            return
+        self.scenario_repl(scenario)
+
+    def scenario_repl(self, scenario):
+        """ REPL for interactive scenario execution """
+        auto_suggest = ScenarioAutoSuggest()
+        example_cli = CommandLineInterface(
+            application=self.create_application(
+                full_layout=False,
+                auto_suggest=auto_suggest,
+                prompt_prefix='(scenario) ',
+                toolbar_hint='In Scenario Mode: Press [Enter] to execute commands   [Ctrl+C]Skip  [Ctrl+D]Quit'
+            ),
+            eventloop=create_eventloop())
+        self.completer.set_scenario_enabled(False)
```

Comment:
```suggestion
        self.completer.enable_scenario_recommendation(False)
```

Reply:
Pleas add comments to explain why we need to disable the scenario recommendation here